tfhe/src/high_level_api/config.rs

```diff
@@ -53,13 +62,21 @@ impl ConfigBuilder {
     pub fn with_custom_parameters<P>(
         block_parameters: P,
         wopbs_block_parameters: Option<crate::shortint::WopbsParameters>,
+        dedicated_compact_public_key_parameters: Option<(
+            crate::shortint::parameters::CompactPublicKeyEncryptionParameters,
+            crate::shortint::parameters::ShortintKeySwitchingParameters,
+        )>,
```

Comment:
I going to do the same remark as for the compression PR,

Is it possible not to add this `dedicated_compact_public_key_parameter` so that the API of this function is not broken
and have something that looks like 

```
Builder::with_custom_param(params, None)
  .with_dedicated_compact_pk_params(...)
```

Reply:
yep just saw your comment on the other PR, I was hesitating, and decided to chameleon my way as the wopbs params, but 100% agree with the proposal

Reply:
Yeah wopbs params are weird, I think I made it that way for wopbs because when creating the Wopbs sks you need the classical and wopbs params at the same time but ideally wopbs should also have been in a dedicated .enable/with